src/Client/ClientBase.cpp

```diff
@@ -2239,7 +2236,17 @@ void ClientBase::runInteractive()
 
     do
     {
+        /// Enable bracketed-paste-mode so that we are able to paste multiline queries as a whole.
+        lr.enableBracketedPaste();
```

Comment:
Ok.

I'd wrap it in a code block
```
{
...
}
```
and use SCOPE_EXIT.

Reply:
Ah, right, maybe it can throw.